Tests/ContainerTest.php

```diff
@@ -1127,4 +1127,23 @@ public function testResolveAliasOfNotSharedKeyFromParentContainer()
 			. ' both the original and the alias should return a similar object.'
 		);
 	}
+
+	/**
+	 * Test that the unique service keys for the container are returned.
+	 *
+	 * @return  void
+	 *
+	 * @since   __DEPLOY_VERSION__
+	 */
+	public function testRetrievingTheContainerKeys()
+	{
+		$this->fixture->set('foo', 'bar');
+		$this->fixture->set('goo', 'car');
+		$this->fixture->alias('boo', 'foo');
```

Comment:
This doesn't seem like good test data because `'foo'` is duplicated on line 1140. This could lead to a false positive in the test results. I suggest using unique values for all constants, perhaps following the [English list of metasyntactic variables](https://en.wikipedia.org/wiki/Metasyntactic_variable#English).

Reply:
This is aliasing the `boo` key to the already set `foo` key, so that `$container->get('boo');` would retrieve the same value as if calling `$container->get('foo');`.  So `boo` and `foo` are unique (separate) keys in the container.

But yes, improvements to the test case are always welcome.

Reply:
Oh right, of course, my mistake. Where the duplication is intentional I prefer to use variables to indicate this; they can also be reused in the assertions.